### ISSUE TITLE
[BUG] Playlist doesn't start from the next episode

### ISSUE THREAD

**Issue by juanma-cvega** [jurialmunkey/plugin.video.themoviedb.helper#997]
### Describe the bug

I've noticed that, when showing the playlist in the OSD created from playing an episode, the list doesn't start from the next episode but from 4 episodes ahead (if there are these many) or from the beginning of the season (not sure what triggers one option ir the other). I would expect this list to contain just the next episodes, and not only from the current season, and to start from the next one. Am I correct in assuming this?

### Steps To Reproduce

1. Go to a TV show.
2. Play an episode.
3. Check the playlist created from it.

### Checklist

- [X] I have searched the issues and this bug has not been reported.
- [X] I have updated TMDbHelper to the most recent version.
- [X] I have described the bug and included steps to reproduce the issue
- [ ] I have uploaded my debug log.

**Comment by jurialmunkey** [comment]
The only way TMDbHelper makes a playlist is if you explicitly set the make_playlist flag in the player and the end plugin is compatible.

Otherwise this is controlled by the end plugin.

**Comment by juanma-cvega** [comment]
Ok, I didn't know the make_playlist flag did that. I just tested it and it works (it shows episodes from the current one to the final episode of the final season). I also tested to play an episode through that playlist but in this case I get the cursor (the episode that is highlighted when accessing the playlist) pointing to a different episode than the one I selected and a playlist with all episodes but only for the current season. I would expect the list to either contain all episodes for the TV show from the current episode being played, like before, or just the same list created in the first play but with the cursor pointing to the current episode. 
A question I have about this behaviour though, assuming I understand correctly how it works, would it be possible to simply load all episodes for the TV show in the playlist from the beginning instead of from the current episode and just put Kodi to play that playlist from the episode selected? That way we would have access to all episodes while playing one of them instead of having to stop the episode to be able to go back to the list and play a previous one. I find myself doing this when playing anime when I'm rewatching an episode from an old show but I don't remember exactly the episode I wanted to watch.
